sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/basicLogicalOperators.scala

```diff
@@ -363,7 +363,8 @@ case class BroadcastHint(child: LogicalPlan) extends UnaryNode {
 }
 
 /**
- * Insert some data into a table.
+ * Insert some data into a table. Note that this plan is unresolved and has to be replaced by the
+ * concrete implementations during analysis.
```

Comment:
To other reviewers, `InsertIntoTable` is an unresolved node for representing INSERT. It will be replaced to `InsertIntoDataSourceCommand`, `InsertIntoHadoopFsRelationCommand` or `InsertIntoHiveTable`.